load-unload.md

```diff
@@ -0,0 +1,74 @@
+# Loading and Unloading Data
+It can be desirable to remove successfully completed workflow runs from the
+Vidarr database. Some reasons include:
+
+- the workflow run was processed incorrectly and needs to be reprocessed
+- the workflow run is no longer useful
+- the workflow run needs to be run with a newer version of the workflow
```

Comment:
Sadly, _workflow run_ is different from _workflow_ though they are confusingly used interchangeably. I will write a glossary.

Reply:
Unfortunately I'm still unclear on this part. You've defined 'workflow run' as 'a particular execution of a workflow version'. That execution is already finished in this case, right? 'The workflow run needs to be run with a newer version' sounds like it would be to create a new workflow run, not do anything to the existing workflow run.